backend/dal/Services/ITenantService.cs

```diff
@@ -0,0 +1,14 @@
+using Pims.Dal.Entities;
+
+namespace Pims.Dal.Services
+{
+    /// <summary>
+    /// ITenantService interface, provides functions to interact with tenants within the datasource.
+    /// </summary>
+    public interface ITenantService : IService
```

Comment:
does it make sense to update the lookup code service api to return the available tenant codes as well?

Reply:
Not really.  We don't plan to have a management tool for this.  The app will only ever be a single tenant and only need to fetch the settings for that tenant.